src/donationswap.py

```diff
@@ -227,9 +222,18 @@ def run_admin_ajax(self, user_secret, command, ip_address, args):
 			logging.error('Ajax Admin Error', exc_info=True)
 			return False, str(e)
 
-	@staticmethod
-	def get_page(name):
-		return util.Template(name).content
+	def get_page(self, name):
+		content = util.Template(name).content
+
+		# This acts as a cache breaker -- just increment
```

Comment:
if i understand this correctly i feel like we could just hash the modified time and use that...
but firstly i'm probably missing something important, and secondly I feel like it really doesn't matter so long as it works :)